api/node.go

```diff
@@ -70,6 +71,7 @@ type Syncer interface {
 type TxAPI interface {
 	AddressExists(types.Address) bool
 	ValidateNonceAndBalance(*types.Transaction) error
+	GetAtxIterByCoinbase(coinbase types.Address) database.Iterator
```

Comment:
I don't think this belongs in the TxAPI interface

Reply:
As mentioned elsewhere I think it can be replaced with something like `GetATXsByCoinbase`